kyuubi-server/pom.xml

```diff
@@ -405,6 +405,19 @@
         </dependency>
     </dependencies>
 
+    <profiles>
+        <profile>
+            <id>spark-3.2</id>
+            <dependencies>
+                <dependency>
+                    <groupId>io.delta</groupId>
+                    <artifactId>delta-storage</artifactId>
```

Comment:
`delta-storage` is the transitive dependency of `delta-core`, do we need to declare it explicitly?